core/src/main/java/org/apache/iceberg/SystemProperties.java

```diff
@@ -41,7 +41,10 @@ private SystemProperties() {}
 
   public static final int IO_MANIFEST_CACHE_MAX_FILEIO_DEFAULT = 8;
 
-  static boolean getBoolean(String systemProperty, boolean defaultValue) {
+  public static final String DELETE_POS_FILES_THREAD_POOL_SIZE =
+      "iceberg.delete-pos.worker.num-threads";
```

Comment:
How about staying with the `iceberg.worker` prefix and adding `iceberg.worker.delete-num-threads` or something? Is that good?